### PR TITLE
v0.16.0

### PR DESCRIPTION
**Pull request by kieckhafer**
# Release Notes for version v0.16.0

## New Features:
Product Search: (PR #1350) (see documentation for more details.) Server-side code is in the package `search-mongo`, client-side & UI is located in `ui-search`.

Shipping: (PR #1357) Fixes to saving errors (#1320), various updates to UI, refactored shipping methods naming conventions (breaking change).

## Changes:
Updated to Meteor 1.4.1.1
Added babelrc and stage-2 preset (#1329)
Move plugin loader from startup to reaction-cli (#1332)
Preserve custom profile fields in Accounts.onCreateUser (#1335)
Copy deprecated cfs:gridfs to local and update npm deps (#1336)
Persistent profile language (#1338)
Only build Docker image on master/development (#1359)
Load default user from env (#1369)
Exclude anonymous users from accounts publication (#1390)
Various UI and dependency updates

## Bug Fixes:
Rounding of order total to negate fractions of cents in shipping invoice

##Testing
Test speed increase (#1355)
Acceptance test setup of user permissions (#1315)

## Contributors
@callmephilip
@priezz

### CODE REVIEW

**Comment by aaronjudd** [comment]
I'm getting 

```
Building dtrace-provider failed with exit code 1 and signal 0
re-run install with environment variable V set to see the build output
---------------
> bcrypt@0.8.7 install /Users/aaronjudd/ongoworks/reaction/node_modules/bcrypt
> node-gyp rebuild
Agreeing to the Xcode/iOS license requires admin privileges, please re-run as root via sudo.
```
^^^ Could be related to new OSX release.
```
npm ERR! bcrypt@0.8.7 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the bcrypt@0.8.7 install script 'node-gyp rebuild'.
```

and 
```
=> Started MongoDB.
W20160920-20:32:54.327(-7)? (STDERR) Note: you are using a pure-JavaScript implementation of bcrypt
```

I feel like this is a regression? I wasn't seeing this in the last couple builds...

**Comment by kieckhafer** [comment]
I agree, the Xcode is probably from Apple,  I was seeing that the other day after I upgraded iTunes, once I updated Xcode all errors went away.

**Comment by zenweasel** [comment]
I didn't upgrade and I am not seeing that error, either in main repo or in a fresh clone

File: imports/plugins/included/search-mongo/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Test Data
```
Comment (zenweasel) [inline imports/plugins/included/search-mongo/README.md]: This file should get removed. My mistake.

File: imports/plugins/included/ui-search/client/templates/searchModal.js [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+          $("body").css("overflow", "visible");
+          Blaze.remove(view);
+        });
+      }
+    };
+  },
+  productSearchResults() {
+    const instance = Template.instance();
+    const results = instance.state.get("productSearchResults");
+    // console.log("productSearchResults", results);
```
Comment (zenweasel) [inline imports/plugins/included/ui-search/client/templates/searchModal.js]: we should probably take these out

**Comment by zenweasel** [comment]
👍

**Comment by jshimko** [comment]
👍 

Fresh clone working well for me.

```sh
reaction init -b development 
cd reaction 
reaction 

# Success.
```

**Pull request merged**